.github/commands.yml

```diff
@@ -51,6 +51,12 @@
             action: 'close',
             comment: "The described behavior is how it is expected to work. If you disagree, please explain what is expected and what is not in more detail. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.\n\nHappy Coding!"
         },
+        {
+            type: 'label',
+            name: '*english-please',
+            action: 'close',
+            comment: "This issue is closed because its non-English descrpition makes it hard for us to understand. Please open a new issue with English description. You might find [Bing Translate](https://www.bing.com/translator) useful."
```

Comment:
This issue is being closed because its description is not in English, that makes it hard for us to work on it. Please open a new issue with an English description. You might find [Bing Translator](https://www.bing.com/translator) useful.

@roblourens Could you take a native-speaker's look?

Reply:
Sounds good to me!

Reply:
@octref Please update and merge. You'll also need to add the label manually. Thanks!

Reply:
Addressed and https://github.com/Microsoft/vscode/labels/%2Aenglish-please